src/_variables.scss

```diff
@@ -1,35 +1,30 @@
-/// Column count to set the ammount of grid columns by default
+/// Default number of grid columns
 /// @type Number
 $column-count: 12 !default;
 
-/// Column prefix to create the classnames for each grid column
+/// Default classname prefix for a grid column
 /// @type String
 $column-prefix: 'aw-col' !default;
 $column-selector: '.#{$column-prefix}' !default;
 
-/// Offset modifier prefix to create the classnames for each grid column offset
+/// Default classname prefix for a grid column offset
 /// @type String
 $offset-modifier: 'offset' !default;
 
-/// Classname delimeters
+/// Default classname delimeters
 /// @type String
 $block-delimiter: '-' !default;
 $element-delimiter: '-' !default;
 $modifier-delimiter: '-' !default;
 
-/// Gutter width to define the space between columns
+/// Default space between columns
 /// @type Length
 $gutter-width: 1rem !default;
 
-/// Outer margin to define the space outside the grid
+/// Default grid outer margin
 /// @type Length
 $outer-margin: 2rem !default;
 
-/// Gutter compensation and half with to define the space between columns
-/// @type Length
```

Comment:
We still need these variables but we have to fix them, they are not supposed to be the same:

```SCSS
$gutter-compensation: (($gutter-width * 0.5) * -1) !default;		
$half-gutter-width: ($gutter-width * 0.5) !default;
```